app/tests/pages/form/[page].test.tsx

```diff
@@ -0,0 +1,140 @@
+import { withRelayOptions } from '../../../pages/form/[id]/success';
+import { screen } from '@testing-library/react';
+import FormPage from '../../../pages/form/[id]/[page]';
+import PageTestingHelper from '../../utils/pageTestingHelper';
+import compiledPageQuery, {
+  PageQuery,
+} from '../../../__generated__/PageQuery.graphql';
+import { NextPageContext } from 'next';
+
+const mockQueryPayload = {
+  Query() {
+    return {
+      applicationByRowId: {
+        status: 'draft',
+        ccbcNumber: 'CCBC-010001',
+        intakeByIntakeId: {
+          ccbcIntakeNumber: 1,
+          closeTimestamp: '2022-09-06T23:59:59-07:00',
+        },
+        projectName: 'Project testing title',
+        updatedAt: '2022-08-15T16:43:28.973734-04:00',
+      },
+      session: {
+        sub: '4e0ac88c-bf05-49ac-948f-7fd53c7a9fd6',
+      },
+    };
+  },
+};
+
+const pageTestingHelper = new PageTestingHelper<PageQuery>({
+  pageComponent: FormPage,
+  compiledQuery: compiledPageQuery,
+  defaultQueryResolver: mockQueryPayload,
+  defaultQueryVariables: {
+    rowId: 1,
+  },
+});
+
+describe('The form page', () => {
+  beforeEach(() => {
+    pageTestingHelper.reinit();
+    pageTestingHelper.setMockRouterValues({
+      query: { id: '1', page: '1' },
+    });
+  });
+
+  it('displays the correct nav links when user is logged in', () => {
+    pageTestingHelper.loadQuery();
+    pageTestingHelper.renderPage();
+
+    expect(screen.getByText('Logout')).toBeInTheDocument();
+  });
+
+  it('does not display the alert or info banner when editing a draft application', () => {
```

Comment:
@matthieu-foucault wasn't sure the best way to test that the banner wasn't there without adding a test-id so I'm checking that the messages aren't there. Let me know if you have a better idea.

Reply:
This looks good to me. This is fine to do here as rendering is synchronous. In Cypress tests, we'd have to make sure that we're not accidentally passing those negative assertions. 

You could consider having the alert texts in constants declared above to ensure consistency between your test cases. This would prevent the scenario where the wording gets updated, you forget to update the wording in this test, and the test below becomes useless.